pkg/chunk/cache/redis_cache.go

```diff
@@ -34,6 +36,8 @@ func (cfg *RedisConfig) RegisterFlagsWithPrefix(prefix, description string, f *f
 	f.DurationVar(&cfg.Expiration, prefix+"redis.expiration", 0, description+"How long keys stay in the redis.")
 	f.IntVar(&cfg.MaxIdleConns, prefix+"redis.max-idle-conns", 80, description+"Maximum number of idle connections in pool.")
 	f.IntVar(&cfg.MaxActiveConns, prefix+"redis.max-active-conns", 0, description+"Maximum number of active connections in pool.")
+	f.StringVar(&cfg.Password, prefix+"redis.password", "", "password to use when connecting to redis.")
```

Comment:
Please add `description+` to the help message and starts the help message with a capital letter, due to how the final message is composed.

_Same feedback applies to the next CLI flag._

Reply:
Thanks @pracucci . Fixed this.